src/main/java/com/dev/cinema/Main.java

```diff
@@ -54,8 +54,10 @@ public static void main(String[] args) throws AuthenticationException {
         user.setEmail("test@test.com");
         user.setPassword("1234");
         userService.add(user);
```

Comment:
you haven't fixed the issue that I suggested to fix(

take a closer look at my comment

Reply:
done